packages/cli/src/util/dev/server.ts

```diff
@@ -1576,7 +1592,7 @@ export default class DevServer {
               const rewriteUrlParsed = new URL(rewritePath);
 
               // `this.address` already has localhost normalized from ip4 and ip6 values
-              const devServerParsed = new URL(this.address);
+              const devServerParsed = this.addressURL;
```

Comment:
You can remove `devServerParsed` now that is only used on this line

Reply:
Good call: [`ae1e1ee` (#8538)](https://github.com/vercel/vercel/pull/8538/commits/ae1e1ee0b3f9d68ebbf71e0036e742d81844c1de)